src/index.css

```diff
@@ -1,13 +1,338 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;800&family=Roboto+Slab:wght@100;200&family=Silkscreen:wght@400;700&display=swap');
+
 body {
+  background-color: #0341be;
+
+  /* background-image: url(https://wallpapercave.com/wp/wp2595119.jpg); */
+
+  color: #fff;
+  width: 100%;
+  font-family: 'Montserrat', sans-serif;
+}
+
+.main-continer {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+nav {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 2rem;
+  width: 100%;
+  font-size: 1rem;
+  padding: 0;
+  padding-left: 1.5rem;
+  padding-top: 1rem;
+  padding-bottom: 0.5rem;
+  background-color: #056cf2;
+}
+
+.home-btn {
+  width: 40px;
+  animation: moviment 2s infinite alternate steps(30);
+}
+
+@keyframes moviment {
+  0% { transform: rotate(25deg); }
+  25% { transform: rotate(0); }
+  75% { transform: rotate(0); }
+  100% { transform: rotate(-25deg); }
+}
+
+.nav-title {
+  width: 100%;
+  display: flex;
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  justify-content: center;
+}
+
+.poke {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  background-color: #0554f2;
+
+  /* background-image: url(https://assets.pokemon.com/assets/cms2/img/misc/virtual-backgrounds/masters/forest.jpg); */
+
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: bottom;
+  width: 100%;
+  height: max-content;
+}
+
```

Comment:
- [ ] Kindly work on the project styling to meet the project requirements as mentioned by the previous reviewer. In order for you to have proper review of your own work, you can open the app on your mobile phone so that you can have a good view of it.